### ISSUE TITLE
Poor Results with ZED Depth Camera

### ISSUE THREAD

**Issue by akeaveny** [j96w/DenseFusion#153]
Thank you for sharing your work @j96w.

I'm able to visualize results for YCB objects using my script [here](https://github.com/akeaveny/DenseFusion/blob/ak/tools/eval_ycb.py). 

![image](https://user-images.githubusercontent.com/56097508/81020800-a58c4400-8e37-11ea-9bea-3f5393920060.png)

However, I have issues when I try to visualize results with a ZED camera using my script [here](https://github.com/akeaveny/DenseFusion/blob/ak/densefusion_ros/src/densefusion_ros.py). I wonder if it's due to the camera intrinsics baked into SegNet which is highlighted in #150.

I've cropped the 720 x 1280 aspect ratio of the ZED camera to 480 x 640. But SegNet returns these classes labels `'Objects: ', array([ 0, 15, 16, 19, 20, 21])` which the soup can has an ID of 4 as per classes.txt. 

```
depth = depth[50:530, 100:740]
rgb = rgb[50:530, 100:740, :] # height x width
```

```
rgb1 = np.transpose(rgb, (2, 0, 1))
rgb1 = norm(torch.from_numpy(rgb1.astype(np.float32))).cuda()
seg_data = self.model(rgb1.unsqueeze(0)) # SEGNET MODEL
seg_data2 = torch.transpose(seg_data[0], 0, 2)
seg_data2 = torch.transpose(seg_data2, 0, 1)
seg_image = torch.argmax(seg_data2, dim=-1)
obj_list = torch.unique(seg_image).detach().cpu().numpy()
label_img = seg_image.detach().cpu().numpy()
print("Objects: ", obj_list)
```

![image](https://user-images.githubusercontent.com/56097508/81021186-98238980-8e38-11ea-81c6-ce2a3df3da8e.png)

**Comment by cyldfs** [comment]
Could you share your visualization code?I failed to open the link.
